offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/product/MaintainProductsView.groovy

```diff
@@ -122,16 +128,21 @@ class MaintainProductsView extends FormLayout {
     }
 
     private void setupPanel(){
+        HorizontalLayout formlayout = new HorizontalLayout()
+        formlayout.setSizeFull()
+        this.addComponent(formlayout)
+
```

Comment:
Why is the `HorizontalLayout` called `formlayout`?
```suggestion
        HorizontalLayout descriptionLayout = new HorizontalLayout()
        descriptionLayout.setSizeFull()
        this.addComponent(descriptionLayout)

```
```suggestion
        FormLayout formLayout = new FormLayout()
        formLayout.setSizeFull()
        this.addComponent(formLayout)

```

Reply:
I considered that but this has error cell and then the formatting looks off. I think there speaks nothing against the horizontal layout

Reply:
Its called form layout because I tried to use it 👀